galley/pkg/server/args.go

```diff
@@ -83,6 +83,15 @@ type Args struct {
 	// allows Galley to start when not all supported CRD are
 	// registered with the kube-apiserver.
 	DisableResourceReadyCheck bool
+
+	// SinkAddress should be set to the address of a MCP Resource
+	// Sink service that Galley will dial out to. Leaving empty disables
+	// sink.
+	SinkAddress string
```

Comment:
Does this address include the `mcp://` scheme prefix, or is it a raw host:port?

Reply:
No, the description in the cli arg --help (root.go) gives an example.

Reply:
(raw host:port)